tc/app/vault/config

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# shellcheck disable=SC1090
+# shellcheck disable=SC1090,SC1091
```

Comment:
delete `SC1090`

Reply:
without SC1090 on this file
![image](https://user-images.githubusercontent.com/50202839/115833560-dbe21b00-a41c-11eb-98e3-fdfb7d3608e7.png)

Reply:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/490917/115844512-93305f00-a428-11eb-99e5-674f7cd607e9.png">
https://github.com/koalaman/shellcheck/wiki/SC1090